server/base/src/main/java/org/apache/accumulo/server/util/ZooZap.java

```diff
@@ -176,6 +178,25 @@ public void execute(String[] args) throws Exception {
 
       }
 
+      if (opts.zapScanServers) {
+        String sserversPath = Constants.ZROOT + "/" + iid + Constants.ZSSERVERS;
+        try {
+          List<String> children = zoo.getChildren(sserversPath);
+          for (String child : children) {
+            message("Deleting " + sserversPath + "/" + child + " from zookeeper", opts);
+
+            var zLockPath = ServiceLock.path(sserversPath + "/" + child);
+            if (!zoo.getChildren(zLockPath.toString()).isEmpty()) {
+              if (!ServiceLock.deleteLock(zoo, zLockPath, "tserver")) {
```

Comment:
Is the `"tserver"` string correct?

Reply:
No, this was a copy paste error. I'm going to modify it to call a different `ServiceLock.deleteLock` method.

Reply:
Fixed in 7521ae6